libs/lib_parsing/Origin.mli

```diff
@@ -70,10 +70,10 @@ type t =
   (* TODO: Evaluate futher using Ppath.t instead of Fpath.t, since it documents
      we want this to be a relative path from the project root. *)
   | GitBlob of {
-      sha : Git_wrapper.sha;
-          (** The hash for this specific git {e blob}. Used for git operations and to
+      blob : OGit_wrapper.hash;
+          (** The sha of the {e blob}  of a blob. Used for git operations and to
```

Comment:
I don't think this makes sense: "the sha of the blob of a blob"? Perhaps just "The sha of this blob."?

Reply:
yea oops